syncapi/consumers/roomserver.go

```diff
@@ -93,6 +93,23 @@ func (s *OutputRoomEventConsumer) Start() error {
 // sync stream position may race and be incorrectly calculated.
 func (s *OutputRoomEventConsumer) onMessage(ctx context.Context, msgs []*nats.Msg) bool {
 	msg := msgs[0] // Guaranteed to exist if onMessage is called
+
+	receivedType := api.OutputType(msg.Header.Get(jetstream.RoomEventType))
+	ok := false
+	// Only handle events we care about
+	for _, eventType := range api.OutputTypes {
```

Comment:
I'd be surprised if we were receiving completely unknown output event types, so in that case `api.OutputTypes` and this loop are probably redundant.

Reply:
The SyncAPI should receive all events, right.